scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpInfo.java

```diff
@@ -114,6 +114,10 @@ default Type outputType() {
 	default int compareTo(final OpInfo that) {
 		if (this.priority() < that.priority()) return 1;
 		if (this.priority() > that.priority()) return -1;
+		int nameComparison = this.toString().compareTo(that.toString());
```

Comment:
It might be nice to think about this a little more, and also to make sure that it aligns with the `equals` method...

Reply:
@hinerm so the one case where we might want this is [here](https://github.com/scijava/incubator/pull/95/files#diff-af5dec84a4906407c0159a8ff35d0858d018c6a57302bb6e9c8d81d901107e47R77) - that test will fail because [`OpEnvironment.infos()`](https://github.com/scijava/incubator/pull/95/files#diff-b271cbe48149d64e6e0bfe70e70eadb7d5d3e3f0d83988a06c2e837af4831103R174-R176) will consider the two `OpInfo`s to be the same.

What if we instead changed the `List<String> names()` method of `OpInfo` to return a `Set<String>`, and then we could check set equality in `compareTo` and `equals`?

Reply:
@hinerm so the one case where we might want this is [here](https://github.com/scijava/incubator/pull/95/files#diff-af5dec84a4906407c0159a8ff35d0858d018c6a57302bb6e9c8d81d901107e47R77) - that test will fail because [`OpEnvironment.infos()`](https://github.com/scijava/incubator/pull/95/files#diff-b271cbe48149d64e6e0bfe70e70eadb7d5d3e3f0d83988a06c2e837af4831103R174-R176) will consider the two `OpInfo`s to be the same.

What if we instead changed the `List<String> names()` method of `OpInfo` to return a `Set<String>`, and then we could check set equality in `compareTo` and `equals`?

Reply:
Fixed with 82367790